provider/resources.go

```diff
@@ -4018,6 +4020,23 @@ func Provider() tfbridge.ProviderInfo {
 
 	fixLabelNames(&prov)
 
+	// managed_zone_id is an ID masquerading as an int64. Since it uses the full 64
+	// bits it has, we need to stringify it to keep more data then a number (float64)
+	// will allow.
+	//
+	// https://github.com/pulumi/pulumi-gcp/issues/1036
+	tfbridge.MustTraverseProperties(&prov, "managed_zone",
```

Comment:
This may slow down startup time. Just FYI. In the case of AWS there are schemas that are hidden behind SchemaFunc() to make them lazy, and traversals of this kind force them to materialize into memory which creates pressure and can cost some significant ms on startup as well as RAM usage. Perhaps not a problem for GCP.

Reply:
I believe Ian added something clever in the bridge which saves the relevant resources in the bridge metadata and only acts on them during startup.

Reply:
Yep. `MustTraverseProperties` does slow startup time during `make tfgen`. It computes a list of effected properties then, and only visits properties that need overrides during normal runtime.

Reply:
Ohh so it's already optimized. Brilliant.